sunbeam-python/sunbeam/plugins/validation/plugin.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import click
+from packaging.version import Version
+from rich.console import Console
+
+from sunbeam.clusterd.service import ClusterServiceUnavailableException
+from sunbeam.commands.terraform import TerraformHelper, TerraformInitStep
+from sunbeam.jobs.common import run_plan
+from sunbeam.jobs.juju import JujuHelper
+from sunbeam.plugins.interface.v1.openstack import (
+    DisableOpenStackApplicationStep,
+    EnableOpenStackApplicationStep,
+    OpenStackControlPlanePlugin,
+    TerraformPlanLocation,
+)
+
+LOG = logging.getLogger(__name__)
+console = Console()
+
+
+class ValidationPlugin(OpenStackControlPlanePlugin):
+    """Deploy tempest to openstack model."""
+
+    version = Version("0.0.1")
```

Comment:
not-blocking: could we convert this to constant

Reply:
Sure